### ISSUE TITLE
[Enhancement] Show time when a post or comment was last edited

### ISSUE THREAD

**Issue by Tywele** [Xyphyn/photon#406]
**Is your feature request related to a problem? Please describe.**
Right now there is no way to see when a post or comment was last edited.

**Describe the solution you'd like**
Show a "Edited x hours/days/weeks ago" somewhere in the post/comment header. Comments at least already show that a comment was edited but not when.

**Comment by sevonj** [comment]
Some options for how to deal with this:
1. Replace post time with last updated time. I think this is done by a few other clients at least.
1. Show the update time as a tooltip when hovering over the pencil icon
1. Show the update time when clicking/tapping the pencil icon (example voyager): 
![image](https://github.com/user-attachments/assets/fe6e3cd3-3b84-4d37-ab50-dfd8cb83853b)
1. Show the update time in actions
![image](https://github.com/user-attachments/assets/f35c9119-d4d5-4bf0-bf11-2e89d161ed43)

I think 2. and 3. options complement each other for a decent experience on both pc and mobile. I think I might like option 4 the best.

So,
- [ ] Expose post update time
- [ ] Expose comment update time 
- [ ] In addition, seems like the pencil icon doesn't appear at all in full post view!
![image](https://github.com/user-attachments/assets/f74e877c-0efe-4ded-a758-cea4ca4f144b)
![image](https://github.com/user-attachments/assets/b57974af-2cef-4d1b-b3ee-b1e81ff089a9)

**Comment by Tywele** [comment]
I think options 2 and 3 would be the best.

**Comment by Xyphyn** [comment]
The pencil icon is the size of my remaining sanity, so I don't think tapping it would be a good trigger. I think I should add detailed stats to the actions on both posts and comments

**Comment by Tywele** [comment]
Please don't only take mobile into consideration. For desktop a hover for the pencil icon would be perfect.

**Comment by sevonj** [comment]
Yeah, I think tooltip is nice. Shouldn't be a problem with mouse cursor accuracy.

**Comment by Xyphyn** [comment]
I'll add both
